skyportal/utils/offset.py

```diff
@@ -228,6 +234,15 @@ def format_hmsdms(skycoord, coord_sep, col_sep):
         "maxname_size": 18,
         "first_line": None,
     },
+    'P200-NGPS': {
+        "coord_sep": ' ',
+        "col_sep": 'XXX',
+        "commentstr": "!",
+        "giveoffsets": False,
+        "maxname_size": 18,
+        "first_line": None,
+        "format": "csv",
```

Comment:
Does this line end up doing anything? Most of the PR is IF statements. Not a problem, just curious.

Reply:
Oh no you are right this is an artifact of my attempt at making it work with the current config-like system and failing. I'll remove it. Thanks!